site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -197,6 +197,20 @@ const CreateConsequenceNetwork = (props: CreateConsequenceNetworkProps): ReactEl
                             buttonClasses="mt-8"
                             isTemplate={isTemplate}
                         />
+
+                        {(props.consequenceIndex || 0) <= 10 && (
+                            <button
+                                formAction={`/api/create-consequence-network${
+                                    isTemplate
+                                        ? "?template=true&addAnotherConsequence=true"
+                                        : "?addAnotherConsequence=true"
+                                }`}
+                                className="govuk-button mt-8 ml-5 govuk-button--secondary"
+                                data-module="govuk-button"
+                            >
+                                Add another consequence
```

Comment:
Undefined exception was observed when creating a new template. Fill the required information and in review page click change link of a consequence to take you to consequence page, click add another consequence and then navigate to type of consequence page and then the selected consequence type page. 

Click cancel button is displaying undefined error

Reply:
done